<?xml version="1.0"?>

<launch>

  <node name="adrv_remote" pkg="adrv_remote" type="adrv_remote.py" output="screen">
    <param name="steer_max" value="0.5" />
    <param name="steer_min" value="-0.5" />
    <param name="drive_max" value="1.0" />
    <param name="drive_min" value="-1.0" />
  </node>

</launch>